sql-native/src/main/java/org/ballerinalang/sql/utils/RecordIteratorUtils.java

```diff
@@ -59,8 +59,6 @@
  * @since 1.2.0
  */
 public class RecordIteratorUtils {
-    private RecordIteratorUtils() {
```

Comment:
Only this constructor is exposed, isn't it? Let's make this constructor public and keep the rest. I will look into the code quality aspects of this

Reply:
`DefaultResultParameterProcessor` this class also exposed

Reply:
Ack.

Reply:
Yes, We can change it to `public`